messaging/src/main/java/org/axonframework/eventhandling/TrackingEventProcessor.java

```diff
@@ -864,6 +865,16 @@ public Throwable getError() {
             return errorState;
         }
 
+        @Override
+        public OptionalLong getReplayPosition() {
+            return trackingToken.position();
```

Comment:
I think this method should check whether `this.trackingToken` is an instance of `ReplayToken`. Otherwise, I wouldn't expect a position.